test/withStyles_test.jsx

```diff
@@ -305,4 +310,44 @@ describe('RTL support', () => {
       expect(resolveNoRTLStub.callCount).to.equal(1);
     });
   });
+
+  describe('contextual create', () => {
+    it('calls ThemedStyleSheet.create without direction set', () => {
+      function MyComponent() {
+        return null;
+      }
+
+      const WrappedComponent = withStyles(() => ({}))(MyComponent);
+      render(<WrappedComponent />);
+      expect(testInterface.create.callCount).to.equal(1);
+    });
+
+    it('calls ThemedStyleSheet.create with LTR direction', () => {
+      function MyComponent() {
+        return null;
+      }
+
+      const WrappedComponent = withStyles(() => ({}))(MyComponent);
+      render(
+        <DirectionProvider direction={DIRECTIONS.LTR}>
+          <WrappedComponent />
+        </DirectionProvider>,
+      );
+      expect(testInterface.create.callCount).to.equal(1);
+    });
+
+    it('calls ThemedStyleSheet.createRTL with RTL direction', () => {
+      function MyComponent() {
+        return null;
+      }
+
+      const WrappedComponent = withStyles(() => ({}))(MyComponent);
+      render(
+        <DirectionProvider direction={DIRECTIONS.RTL}>
+          <WrappedComponent />
+        </DirectionProvider>,
+      );
+      expect(testInterface.createRTL.callCount).to.equal(1);
```

Comment:
`expect(testInterface.createRTL).to.have.property('callCount', 1)` will have a nicer failure message.